src/toil/common.py

```diff
@@ -563,10 +563,12 @@ def addOptions(parser: ArgumentParser, config: Optional[Config] = None) -> None:
     move_exports.add_argument("--moveExports", dest="moveExports", action='store_true', help=move_exports_help)
     move_exports.add_argument("--noMoveExports", dest="moveExports", action='store_false', help=move_exports_help)
     move_exports.set_defaults(moveExports=False)
-    file_store_options.add_argument('--disableCaching', dest='disableCaching', action='store_true',
-                                    default=False,
-                                    help='Disables caching in the file store. This flag must be set to use '
-                                         'a batch system that does not support cleanup, such as Parasol.')
+
+    caching = file_store_options.add_mutually_exclusive_group()
+    caching_help = ("Enable or disable caching for your workflow, specifying this overrides default from job store")
+    caching.add_argument('--disableCaching', dest='caching', action='store_false', help=caching_help)
+    caching.add_argument('--caching', dest='caching', type=bool, help=caching_help)
```

Comment:
I saw where you put a `--caching=FALSE` example, and I was worried that `"FALSE"` might not parse as `False` because of the case, so I tested it:

```
Python 3.8.10 (default, Nov 14 2022, 12:59:47) 
[GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> bool('FALSE')
True
```

It turns out that `type=bool` just doesn't work properly, [because `bool()` of any non-empty string is `True`](https://stackoverflow.com/q/15008758).

I think we actually need to do something more like https://stackoverflow.com/a/59579733

```suggestion
    from distutils.util import strtobool
    caching.add_argument('--caching', dest='caching', type=lambda val: bool(strtobool(val)), help=caching_help)
```

Reply:
This also suggests that maybe we could use a test for the parsing? I'm not sure the best way to do that though.